site/docs/hive.md

```diff
@@ -98,8 +98,8 @@ To globally register different catalogs, set the following Hadoop configurations
 
 | Config Key                                    | Description                                            |
 | --------------------------------------------- | ------------------------------------------------------ |
-| iceberg.catalog.<catalog_name\>.type          | type of catalog: `hive` or `hadoop`                    |
-| iceberg.catalog.<catalog_name\>.catalog-impl  | catalog implementation, must not be null if type is null |
+| iceberg.catalog.<catalog_name\>.type          | type of catalog: `hive`, `hadoop` or empty if `catalog-impl` will be set.                  |
```

Comment:
I don't think it is necessary since this is the original intention to leave the `type` empty in case the `catalog-impl ` will be set.

Reply:
Non-blocking notes:
1) I agree that this can mostly be rolled back. I would say that `type` could be "hive or hadoop, or left unset if using a custom catalog".